lang/cpp/reef-bridge-clr/src/main/Cpp/CppBridge/JavaClrBridge/JavaClrBridge.sln

```diff
@@ -1,14 +1,14 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
lang/cpp/reef-bridge-clr/src/main/Cpp/CppBridge/JavaClrBridge/JavaClrBridge.sln should be renamed as 
lang/cpp/reef-bridge-clr/src/main/Cpp/CppBridge/JavaClrBridge/Org.Apache.REEF.Bridge.JavaClrBridge.sln